packages/bruno-app/src/components/Sidebar/ImportCollection/index.js

```diff
@@ -42,6 +51,9 @@ const ImportCollection = ({ onClose, handleSubmit }) => {
         <div className="text-link hover:underline cursor-pointer mt-2" onClick={handleImportInsomniaCollection}>
           Insomnia Collection
         </div>
+        <div className="text-link hover:underline cursor-pointer mt-2" onClick={handleImportOpenapiCollection}>
+          OpenAPI Collection
```

Comment:
More precise it is OpenAPI Specification

Reply:
Happy to add v3 here if that is the consensus. The question here is, should there be separate options for v2 and v3 openapi specs in the Import list? 

Pros for current approach:
- unburden user with version selection, as it can be read from the spec. let Bruno just handle it (eventually will support v2 as well)
- Fewer list options in Import modal

Cons:
- perhaps unclear to the user whether their Openapi spec is supported
- Less visibly declarative as to version being imported (reliant on error reporting on import validation)

Will defer to decision from community!

Reply:
If it's not to ugly maybe we could show which version are supported: 
"OpenAPI Specification V3"
and then later on:
"OpenAPI Specification V2 & V3"

That way we could have Bruno handle it and we tell the users early on if they will have success in importing their spec fil.